main.js

```diff
@@ -8,9 +8,51 @@
         attribution: '&amp;copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
     }).addTo(map);
 
+    //Array of markers and a parallel array that holds their date of occurance
+    var markers = new Array();
+    var dates = new Array();
+
     //WPRDC data
     const WPRDC_BASE_URL = 'https://data.wprdc.org/api/action/datastore_search_sql?sql=';
 
+    //Create a new Date object for the current date
+    var currentDate = new Date();
+
+    //The following are functions that display records created within 1, 7, and
+    //30 days respectively
+    function displayToday() {
+        markers.forEach((marker, i) => {
+            if (dates[i][0] == currentDate.getFullYear() && dates[i][1] == currentDate.getMonth() + 1 && dates[i][2] == currentDate.getDate()) {
```

Comment:
This loop still works with the new object system, except that instead of referencing `dates[i][j]`, it would be `marker.incidentYear`, `marker.incidentMonth`, etc. Much clearer and all in the same marker object, so it's more functional.